lib/wed/object_check.js

```diff
@@ -68,7 +68,7 @@ function _required(template, name) {
  *
  * @param {Object} template The template to use for the check.
  * @param {Object} object The object to check
- * @returns {Object.<string, Array.<string>>}} The results.
+ * @returns {Object.<string>, Array.<string>} The results.
```

Comment:
I'm pretty sure the original was correct. The function returns objects which have for keys strings and for values arrays of strings. And I don't think `{Object.<string>, Array.<string>}` is meaningful. `@returns` optionally takes a type. When the type is present it is enclosed in curly braces: these braces do not take part in specifying what the type actually is. So the notation you have would mean that the function returns something of type `Object.<string>, Array.<string>` but I don't see how this makes sense as a type.